src/plugins/vis_types/timeseries/server/lib/search_strategies/capabilities/rollup_search_capabilities.ts

```diff
@@ -59,13 +59,10 @@ export class RollupSearchCapabilities extends DefaultSearchCapabilities {
         return acc;
       }, this.createUiRestriction({}) as Record<string, boolean>);
 
-    return Object.keys(this.availableMetrics).reduce(
-      (acc, item) => ({
-        ...acc,
-        [item]: getFields(this.availableMetrics[item]),
-      }),
-      baseRestrictions
-    );
+    return Object.keys(this.availableMetrics).reduce((acc, item) => {
+      (acc as Record<string, unknown>)[item] = getFields(this.availableMetrics[item]);
+      return acc;
+    }, baseRestrictions);
```

Comment:
nit: I would probably take the opportunity to rewrite this with a `for` loop. WDYT?

Reply:
Same as previous comment